src/appmixer/intelswift/core/SaveSyncId/SaveSyncId.js

```
@@ -0,0 +1,29 @@
+    async receive(context) {
+
+        const { platformType, contactId, tenantId, botId, origin, syncId } = context.messages.in.content;
+
+        let body = {};
+        if(!platformType) throw 'platformType not found';
+
+        if(platformType == 'zoho') body.zohoId = syncId;
+        if(platformType == 'hubspot') body.hubspotId = syncId;
+        if(platformType == 'pipedrive') body.pipedriveId = syncId;
+
+        const { data } = await makeRequest(context, `/atombot/api/v1/contacts/update/?tenantId=${tenantId}&contactId=${contactId}`, { 
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)